File: src/CloudIntegrations.java
import com.google.cloud.storage.Storage;
import com.google.cloud.storage.StorageOptions;
import software.amazon.awssdk.services.s3.S3Client;
import software.amazon.awssdk.services.s3.model.GetObjectRequest;

final class CloudIntegrations {
  void readAssets() {
    S3Client s3 = S3Client.builder().build();
    s3.getObject(GetObjectRequest.builder().bucket("repello-java-assets").build());

    Storage storage = StorageOptions.newBuilder().build().getService();
    storage.get("repello-java-training-data");
  }
}

File: src/CloudIntegrations.cs
using Amazon.SageMakerRuntime;
using Amazon.SageMakerRuntime.Model;
using Google.Cloud.Storage.V1;

internal sealed class CloudIntegrations
{
    public void ReadAssets()
    {
        var runtime = new AmazonSageMakerRuntimeClient();
        runtime.InvokeEndpointAsync(new InvokeEndpointRequest
        {
            EndpointName = "repello-csharp-inference"
        });

        var storage = StorageClient.Create();
        storage.GetBucket("repello-csharp-training-data");
    }
}
